public/modules/home/views/partials/header.html

```diff
@@ -7,7 +7,8 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="/">{{ brand }}</a>
+      <a class="navbar-brand visible-xs" href="/">{{ brandSmall }}</a>
+      <a class="navbar-brand hidden-xs" href="/">{{ brand }}</a>
```

Comment:
This appears to be bigger in the keynote snapshot, maybe increase the font one size or two.